### PR TITLE
French improvement

### PR DESCRIPTION
**Pull request by ShiiFu**
Hi team,

I did some minor modifications about : 
- french translation
- french documentation : fix old broken link

Hope this fast reviewing of french parts will help :)

The french documentation is actually incomplete, I can help to update it.

### CODE REVIEW

File: docs/fr/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Cette documentation est divisée en plusieurs parties :
 
 * La [documentation utilisateurs](./users/02_First_steps.md) pour découvrir
   les fonctionnalités de FreshRSS.
-* La [documentation administrateurs](../en/admins/01_Installation.md) (en
+* La [documentation administrateurs](../en/admins/01_Index.html) (en
```
Comment (Frenzie) [inline docs/fr/index.md:32]: In principle this file is auto-generated based on the POT file but I think po4a should also pick up on the reverse.

https://github.com/FreshRSS/FreshRSS/blob/2b26f9dcbde3cf2f2657b33cf85245577cbacce0/docs/i18n/freshrss.fr.po#L3914-L3921

**Review by Frenzie — APPROVED** [review]
> The french documentation is actually incomplete, I can help to update it.

Please, have at it! :-)

**Comment by Alkarex** [comment]
Merci @ShiiFu 
Could you please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md ?

**Comment by ShiiFu** [comment]
Sure, I will do it with the next doc translation

**Pull request merged**